qiskit/aqua/operators/state_fns/circuit_state_fn.py

```diff
@@ -354,6 +359,9 @@ def reduce(self) -> OperatorBase:
                     del self.primitive.data[i]
         return self
 
+    def expand_with_identity(self, num_qubits: int) -> 'CircuitStateFn':
+        return self.permute(list(range(num_qubits, num_qubits + self.num_qubits)))
```

Comment:
Nice one, but a bit confusing to read. Could you add a comment? 
Also: Is this the order? Meaning should the identities be padded on top or bottom of the circuit?

Reply:
Sure, good point, it deserves commenting, that we are reversing qiskit endinanness here, just like in ```tensor``` method in CircuitOp.
So this is just ```self.tensor(identity)```.